ERROR - 2023-03-20 22:19:04 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
#5 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(41): countData('datauser')
#6 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(213): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#8 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashb...', Array, true)
#9 C:\xampp\htdocs\activityreport\app\Controllers\Dashboard.php(24): view('dashboard/dashb...', Array)
#10 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\activityreport\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
#4 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(41): countData('datauser')
#5 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(213): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashb...', Array, true)
#8 C:\xampp\htdocs\activityreport\app\Controllers\Dashboard.php(24): view('dashboard/dashb...', Array)
#9 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Dashboard->index()
#10 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\activityreport\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2023-03-20 22:19:04 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `datauser`', [], false)
 3 APPPATH\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
 4 APPPATH\Views\dashboard\dashboard.php(41): countData('datauser')
 5 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\dashboard\\dashboard.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashboard', [], true)
 8 APPPATH\Controllers\Dashboard.php(24): view('dashboard/dashboard', [...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-20 22:19:12 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
#5 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(41): countData('datauser')
#6 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(213): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#8 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashb...', Array, true)
#9 C:\xampp\htdocs\activityreport\app\Controllers\Dashboard.php(24): view('dashboard/dashb...', Array)
#10 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Dashboard->index()
#11 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#12 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\activityreport\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
#4 C:\xampp\htdocs\activityreport\app\Views\dashboard\dashboard.php(41): countData('datauser')
#5 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(213): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashb...', Array, true)
#8 C:\xampp\htdocs\activityreport\app\Controllers\Dashboard.php(24): view('dashboard/dashb...', Array)
#9 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Dashboard->index()
#10 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\activityreport\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2023-03-20 22:19:12 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `datauser`', [], false)
 3 APPPATH\Views\dashboard\dashboard.php(38): CodeIgniter\Database\BaseBuilder->countAllResults()
 4 APPPATH\Views\dashboard\dashboard.php(41): countData('datauser')
 5 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\dashboard\\dashboard.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashboard', [], true)
 8 APPPATH\Controllers\Dashboard.php(24): view('dashboard/dashboard', [...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:44:56 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:45:29 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:47:08 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:48:28 --> Call to undefined function countData()
in APPPATH\Views\dashboard\dashboard.php on line 50.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(24): view('dashboard/dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-20 22:49:38 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\activityreport\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\activityreport\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#8 C:\xampp\htdocs\activityreport\vendor\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#9 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->attemptLogin()
#10 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\activityreport\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\activityreport\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2023-03-20 22:49:38 --> Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datauser`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 4 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
 5 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate([...], true)
 6 VENDORPATH\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt([...], false)
 7 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->attemptLogin()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:50:34 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:52:44 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:52:46 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 22:52:48 --> Call to undefined method App\Models\UserModel::logActivationAttempt
in SYSTEMPATH\Model.php on line 842.
 1 VENDORPATH\myth\auth\src\Controllers\AuthController.php(342): CodeIgniter\Model->__call('logActivationAttempt', [...])
 2 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->activateAccount()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:01:58 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Dashboard.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Dashboard->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:06:38 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Datauser.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Datauser->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:09:04 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Datauser.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Datauser->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:09:08 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Datauser.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Datauser->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:10:30 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Datauser.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Datauser->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:10:42 --> Call to a member function getUser() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:11:07 --> Call to a member function getUsers() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:12:10 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\datauser\home.php on line 47.
 1 APPPATH\Views\datauser\home.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'APPPATH\\Views\\datauser\\home.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\datauser\\home.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('datauser/home', [], true)
 5 APPPATH\Controllers\Datauser.php(20): view('datauser/home', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:12:15 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\datauser\home.php on line 47.
 1 APPPATH\Views\datauser\home.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'APPPATH\\Views\\datauser\\home.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\datauser\\home.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('datauser/home', [], true)
 5 APPPATH\Controllers\Datauser.php(20): view('datauser/home', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:14:29 --> Call to a member function getUsers() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:14:43 --> Call to a member function getUser() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:14:45 --> Call to a member function getUser() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:14:46 --> Call to a member function getUser() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:15:12 --> Call to a member function getUser() on null
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:16:35 --> Object of class App\Controllers\Datauser could not be converted to string
in APPPATH\Controllers\Datauser.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:17:15 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\datauser\home.php on line 47.
 1 APPPATH\Views\datauser\home.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'APPPATH\\Views\\datauser\\home.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\datauser\\home.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('datauser/home', [], true)
 5 APPPATH\Controllers\Datauser.php(20): view('datauser/home', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:20:28 --> foreach() argument must be of type array|object, string given
in APPPATH\Views\datauser\home.php on line 47.
 1 APPPATH\Views\datauser\home.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, string given', 'APPPATH\\Views\\datauser\\home.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\datauser\\home.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('datauser/home', [], true)
 5 APPPATH\Controllers\Datauser.php(20): view('datauser/home', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:21:05 --> Undefined variable $data
in APPPATH\Views\datauser\home.php on line 47.
 1 APPPATH\Views\datauser\home.php(47): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', 'APPPATH\\Views\\datauser\\home.php', 47)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\datauser\\home.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('datauser/home', [], true)
 5 APPPATH\Controllers\Datauser.php(20): view('datauser/home', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Datauser->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Datauser))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-20 23:21:32 --> Class "App\Models\UserModel" not found
in APPPATH\Controllers\Datauser.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Datauser->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
